### ISSUE TITLE
Lead conversion: choose assignment per new dataset

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2334]
#### Issue

Suggestion for usability and efficiency during lead conversion.
#### Expected Behavior

Ability to choose which user to assign datasets to, that are created during lead conversion.
#### Actual Behavior

All new datasets created during lead conversion process get assigned automatically to same user as the current lead owner.
#### Suggestion

Show relationship field "assign to" in the lead conversion screen for all available datasets. (Contact, Account, Opportunity, Call, Meeting, Task, Note) Prefill with current lead ownerm but enable changes to be made during conversion process. 

This way a task or call (e.g.) could be planned and assigned to a different user (in same team, let's say) in a single step during lead conversion.
#### Your Environment

- SuiteCRM Version used: SuiteCRM 7.7.4

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Lead conversion: choose assignment per new dataset](https://trello.com/c/JOa6BQTw/68-lead-conversion-choose-assignment-per-new-dataset)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
